### PR TITLE
Add multinode role-adding instructions

### PR DESCRIPTION
**Pull request by WireBaron**
This change adds a section to the basic multinode setup document
that describes how to add roles to a permissive multinode setup.

It also updates several distributed_exec invocations to use CALL
instead of select.

### CODE REVIEW

File: getting-started/setup-multi-node-basic.md
Comment (solugebefola) [inline getting-started/setup-multi-node-basic.md:86]:
```suggestion
but only if running as a superuser common to all nodes. In order to allow additional
```

File: getting-started/setup-multi-node-basic.md
Comment (solugebefola) [inline getting-started/setup-multi-node-basic.md:107]:
```suggestion
It's important that the role be created with LOGIN permission on the data nodes, even if it
```

**Review by solugebefola — APPROVED** [review]

**Review by k-rus — APPROVED** [review]

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md]: Fixed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md]: Fixed

File: getting-started/setup-multi-node-auth.md
Comment (k-rus) [inline getting-started/setup-multi-node-auth.md:45]: May be to move this section last?

File: getting-started/setup-multi-node-basic.md
Comment (k-rus) [inline getting-started/setup-multi-node-basic.md:44]: This section feels odd for Forge and Cloud users, who will stay in the document. I understand that it is for self-managed users who will jump into this file for the add data nodes instructions.

File: getting-started/setup-multi-node-auth.md
Comment (k-rus) [inline getting-started/setup-multi-node-auth.md:134]: I suggest to remove reference to trust authentication. IMHO it is unlikely the reader will read the trust authentication first and then the password authentication.
```suggestion
Password authentication is
```

Comment (k-rus) [reply getting-started/setup-multi-node-auth.md:134]: I guess it is necessary to repeat how to find the hba file.

File: getting-started/setup-multi-node-auth.md
Comment (k-rus) [inline getting-started/setup-multi-node-auth.md:175]: I don't think it is needed to refer to the trust authentication, since all commands are given here.

File: getting-started/setup-multi-node-auth.md
Comment (k-rus) [inline getting-started/setup-multi-node-auth.md:404]: Does the reference to trust authentication mean that it should be a common description of adding a user?

**Review by k-rus — COMMENTED** [review]
Great start! I have number of comments and suggestions

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: Removed reference and added a note about using `SHOW hba_file`.

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md:146]: We should probably add this once we have our GUC document merged so we can properly link to it.

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: Good point, removed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md]: Fixed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md:44]: Agreed, but it is important for the self-managed users, and hopefully we'll have a way to add users in Forge eventually.  At that point, this section can be expanded to explain how to add setup those roles for multinode.

File: getting-started/setup-multi-node-auth.md
Comment (k-rus) [reply getting-started/setup-multi-node-auth.md:15]: To my knowldege @feikesteenbergen and the team created different Dockerfiles for AN and DN and they use different settings there. I haven't looked what exactly is different.

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [inline getting-started/setup-multi-node-auth.md:31]:
```suggestion
## Node-to-node communication [](#node-communication)
```

File: api.md
Comment (mfreed) [inline api.md:937]: Heads up - I think @k-rus has a separate PR which changes this.  Might want to remove this change if there are other changes to avoid the conflict.

https://github.com/timescale/docs.timescale.com-content/pull/569

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [inline getting-started/setup-multi-node-auth.md:12]: Does this belong on a page around "_authentication_"?  Just asking if this is right place here?  (Also, what does timescaledb-tune configure it too?)

if tune configures, do we need this?  And instead add a mention of this to the main configuration page (which has a section about important GUCs _for postgres) and/or troubleshooting?

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [reply getting-started/setup-multi-node-auth.md:111]: (Agree to move to subsequent discussion)

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [inline getting-started/setup-multi-node-auth.md:133]: Should this really be a separate subsection of step 1?  Might be confusing -> isn't it just part of Step 1?  (Or is it a new step 2?)

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [inline getting-started/setup-multi-node-auth.md:186]:
```suggestion
Finally, use [`distributed_exec`][distributed_exec] to add the role to all of the data nodes. In addition to LOGIN,
```

Link these type of things to API docs?

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [inline getting-started/setup-multi-node-auth.md:393]:
```suggestion
#### 5. Attach the data nodes to the access node
```

File: getting-started/setup-multi-node-auth.md
Comment (mfreed) [inline getting-started/setup-multi-node-auth.md:416]:
```suggestion
And finally add the role to all of the data nodes with [`distributed_exec`][distributed_exec]:
```

File: getting-started/setup-multi-node-basic.md
Comment (mfreed) [inline getting-started/setup-multi-node-basic.md:14]:
```suggestion
TimescaleDB multi-node can be created as part of a self-managed deployment
or (coming soon) as a managed cloud deployment.  In order to set up a
self-managed cluster, including how to configure the nodes for secure
communication and creating users/roles across servers, please follow
[these instructions][advanced setup].
```

File: getting-started/setup-multi-node-basic.md
Comment (mfreed) [inline getting-started/setup-multi-node-basic.md:19]:
```suggestion
In the case of Timescale Cloud and Forge, the created services already contain
PostgreSQL with TimescaleDB loaded and the created user `tsdbadmin` as superuser.
In this case, all you will need to do is decide which service should be the access
node, and follow the instructions in the next section.  More information will be
forthcoming as TimescaleDB multi-node is made available on these cloud platforms.
```

File: getting-started/setup-multi-node-basic.md
Comment (mfreed) [inline getting-started/setup-multi-node-basic.md:37]: Can hold off until feature launched/available.
```suggestion
but only if running as a superuser common to all nodes (e.g., by default,
the `postgres` user).
```

File: getting-started/setup-multi-node-basic.md
Comment (mfreed) [inline getting-started/setup-multi-node-basic.md:67]: Minor point, but should we rename these:

`getting-started/setup-multi-node` and `getting-started/setup-multi-node/auth` ?

File: page-index/page-index.js
Comment (mfreed) [inline page-index/page-index.js:136]: This is a mouthful.  Maybe just "Securing Access" or something? Or just keep as "Configuring access" or "Configuring authentication"?  Or "Advanced setup" ? Reader knows it's about multi-node already given hierarchy"

File: tutorials/clustering.md
Comment (mfreed) [inline tutorials/clustering.md:170]: @k-rus other PR might have handled this as well..

**Review by mfreed — APPROVED** [review]
A number of comments that should be addressed, but afterwards, LGTM.

File: api.md
Comment (WireBaron) [reply api.md]: Yeah, I see he's merged that change so I'll make sure to iron out all conflicts before merging.

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md:12]: I struggled a bit with this section.  The `max_prepared_transactions` is required for multi-node, and is not set by timescaledb-tune.  So really this section only exists to tell users to set this.

I think mentioning tune here is a bit confusing, I'll just change this to [normal configuration for TimescaleDB][link to configuration page].

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: fixed

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: I've decided to make this a new step 2.

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: Fixed here and in the certificate section

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: Fixed

File: getting-started/setup-multi-node-auth.md
Comment (WireBaron) [reply getting-started/setup-multi-node-auth.md]: fixed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md]: changed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md]: changed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md]: fixed

File: getting-started/setup-multi-node-basic.md
Comment (WireBaron) [reply getting-started/setup-multi-node-basic.md:71]: It looks like there's already a zombie setup-multi-node page still lingering in this directory that was split into these two new pages (setup-multi-node-basic and setup-multi-node-auth).  @solugebefola did this refactoring about three weeks back, so I'd like to get his feedback here before mucking things up too much.

File: page-index/page-index.js
Comment (WireBaron) [reply page-index/page-index.js]: None of these really seem to quite capture the content, but "Advanced Setup" does seem like an improvement over the current title.  I'll just go with that.

File: tutorials/clustering.md
Comment (WireBaron) [reply tutorials/clustering.md]: Looks like this one wasn't in Ruslan's PR.

**Pull request merged**
